### PR TITLE
feature/MTM-46344-count-of-device-in-group-incorrect-docs-update

### PR DESCRIPTION
**Pull request by Airdroplet**
* added short explanation on subassets count displayed in subassets view of assets navigatior

### CODE REVIEW

File: content/users-guide/cockpit-bundle/managing-assets.md [hunk 1]
```diff
@@ -86,6 +86,10 @@ Moreover, subassets are shown in the **Subassets** tab of the particular group w
 
 <img src="/images/users-guide/cockpit/cockpit-groups-subassets.png" name="Subassets"/>
 
+{{< c8y-admon-info >}}
+Subassets count displayed on top of the table of **Subassets** tab is showing total number of child assets assigned to current group. Any type of managed object can be a child asset. For more details on counting of objects please check OpenAPI specification for "[retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource)"
```
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/users-guide/cockpit-bundle/managing-assets.md:90]: This is actually supposed to work by using the standard URL (without version). @rssag @manasesjesus Any update here? Who is taking care that this gets implemented?

[the remark this answers is not in the document]
Comment (rssag) [reply content/users-guide/cockpit-bundle/managing-assets.md:90]: suggestion for the time being would be to use the standard URL variable, then the current version variable.

i notified jacek and martin about the need for another server URL.

Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/managing-assets.md:90]:
```suggestion
The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```

[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/cockpit-bundle/managing-assets.md]: where do you have description for standard variables which can be used in documentation? I didn't find anything in README.md

File: content/users-guide/cockpit-bundle/managing-assets.md [hunk 2]
```diff
@@ -86,6 +86,10 @@ Moreover, subassets are shown in the **Subassets** tab of the particular group w
 
 <img src="/images/users-guide/cockpit/cockpit-groups-subassets.png" name="Subassets"/>
 
+{{< c8y-admon-info >}}
+The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://cumulocity.com/api/10.11.0/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```
Comment (rssag) [inline content/users-guide/cockpit-bundle/managing-assets.md:90]:
```suggestion
The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```

Comment (Airdroplet) [reply content/users-guide/cockpit-bundle/managing-assets.md]: ok

File: content/users-guide/cockpit-bundle/managing-assets.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/cockpit-bundle/managing-assets.md]: applied

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/cockpit-bundle/managing-assets.md [hunk 3]
```diff
@@ -86,6 +86,10 @@ Moreover, subassets are shown in the **Subassets** tab of the particular group w
 
 <img src="/images/users-guide/cockpit/cockpit-groups-subassets.png" name="Subassets"/>
 
+{{< c8y-admon-info >}}
+The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to [retrieve all child assets of a specific managed object](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```
Comment (manasesjesus) [inline content/users-guide/cockpit-bundle/managing-assets.md:90]:
```suggestion
The count displayed on top of the table on the **Subassets** tab shows the total number of child assets assigned to the current group. Any type of managed object can be a child asset. For more details on the counting of objects refer to the operation [Retrieve all child assets of a specific managed object](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#operation/getManagedObjectChildAssetsResource) in the {{< openapi >}}.
```

Comment (Airdroplet) [reply content/users-guide/cockpit-bundle/managing-assets.md]: ok

**Review by manasesjesus — APPROVED** [review]

**Review by pawel-rynarzewski-sag — APPROVED** [review]
@BeateRixen, please merge it.

**Pull request merged**